lib/bolt/cli.rb

```diff
@@ -413,7 +417,14 @@ def list_plans
 
     def list_targets
       update_targets(options)
-      outputter.print_targets(options)
+      targets = options[:targets].map(&:name)
+      outputter.print_targets(targets)
+    end
+
+    def show_targets
+      update_targets(options)
+      targets = inventory.targets.values.map(&:to_h)
```

Comment:
This line also seems like it belongs in the outputter

Reply:
Updated.